src/views/AboutUs/aboutus.scss

```diff
@@ -1,8 +1,6 @@
-$bgcolor: #fff;
 $fontcolor: #676666;
 $headingcolor: #6d6666;
```

Comment:
Why do we still have these variables here?

Reply:
Same as above. the variables are specific to the About component.

Reply:
Ok